testing/asserts_test.ts

```diff
@@ -1342,13 +1343,14 @@ Deno.test({
 
     let TestClassWithSameName: new () => unknown;
     {
-      class TestClass1 {}
-      TestClassWithSameName = TestClass1;
+      class TestClass3 {}
```

Comment:
So swc renames this class to `TestClass31` because it's in the same scope as `TestClass3`. I will open an issue tomorrow to look into this.